content/md/franklin-insights/franklin-insights-getting-started.md

```diff
@@ -0,0 +1,39 @@
+---
+id: franklin-insights-getting-started
+themes: franklin-insights
+title:  Franklin Insights **getting started**
+ee-only: true
+related: franklin-insights-mappings, franklin-insights-subscriptions
+---
+
+# Overview
```

Comment:
what about adding a dedicated paragraph to explain catalog edit impacts (ex. delete an attribute used as an identifier)